tests/model_indexes/tests.py

```diff
@@ -129,6 +147,23 @@ def test_deconstruct_with_condition(self):
             }
         )
 
+    def test_deconstruct_with_function_field(self):
+        index = models.Index(
+            name='book_func_idx',
+            fields=[Lower('title'), 'author'],
```

Comment:
Could you add a corresponding test for the creation and order of the expressions for mixed function and columns? It should be supported on at least PostgreSQL. I'm not sure if mixed column/function indexes are supported on other backends.

Reply:
Is this not what `tests.schema.tests.SchemaTests.test_func_index_field_and_expression` are testing?

Reply:
Ah yes. Sorry for missing this.